app/response_processor.py

```diff
@@ -120,6 +138,21 @@ def send_receipt(self, decrypted_json):
 
         self.logger.info("Receipt published")
 
+    def send_notification(self, survey_id):
+
+        try:
+            if survey_id == 'census':
+                self.logger.info("Ignoring received CTP submission")
+            elif survey_id == '144':
+                self.logger.info("About to publish notification to cora queue")
+                self.cora_notifications.publish_message(self.tx_id)
+            else:
```

Comment:
I'd be more explicit about routing - don't let common software just be a catch all. If the survey is not known it should raise an exception (otherwise it's only going to break when it gets to the cs transformer)

Reply:
If the survey is not known it gets caught in sdx-validate and the response is marked as invalid. Invalid responses do not send notifications to downstream.